data/filedefs/filetypes.Arduino.conf

```diff
@@ -0,0 +1,56 @@
+# Arduino custom filetype
+#
+# By SukkoPera <software@sukkology.net>
+# 2016/12/14
```

Comment:
I put the date there as I plan to maintain the filetype on my own and submit it upstream from time to time. This way I can see immediately if I did any updates but well, I'll just remove this header, as it seems to cause more concerns than benefits :).